runtime/src/engine.rs

```diff
@@ -136,6 +136,17 @@ impl Marine {
             .and_then(|module| module.export_record_type_by_id(record_id))
     }
 
+    /// Returns a heap size that all modules consume in bytes.
+    pub fn heap_statistic(&self) -> HeapStatistic<'_> {
```

Comment:
Is it an expensive operation or a cheap one? 

IMO it's a very important question for API, can you please mention that in the comment.

Reply:
It's a very cheap operation, I'll add a comment